vnext/DeforkingPatches/ReactCommon/cxxreact/Instance.cpp

```diff
@@ -94,10 +91,14 @@ void Instance::setSourceURL(std::string sourceURL) {
   nativeToJsBridge_->loadApplication(nullptr, nullptr, std::move(sourceURL));
 }
 
-void Instance::loadScriptFromString(std::unique_ptr<const JSBigString> bundleString,
-                                    std::string bundleURL, // TODO(OSS Candidate ISS#2710739)
-                                    bool loadSynchronously) {
-  SystraceSection s("Instance::loadScriptFromString", "bundleURL", bundleURL); // TODO(OSS Candidate ISS#2710739)
+void Instance::loadScriptFromString(
+    std::unique_ptr<const JSBigString> bundleString,
+    std::string bundleURL, // TODO(OSS Candidate ISS#2710739)
```

Comment:
Oh fun. It looks like "yarn format" didn't pick up the clang format file until after it was committed. We're still not matching FB style anyway, so I can add a new iteration with unformated files if that makes diffing easier.

Reply:
Is this issues with our format script not handling multiple clang format files (entirely possible)

Reply:
The script does pass `-style=file` which is documented to ask clanf-format to look for the closest parent directory of the input file.

It does seem like we're doing the wrong thing here though with `-assume-filename` based on the documentation I found.

``` json
"scripts": {
    ...
    "format": "node packages/scripts/formatFiles.js -i -style=file -assume-filename=../.clang-format",
    "format:verify": "node packages/scripts/formatFiles.js -i -style=file -assume-filename=../.clang-format -verify"
  },
```

> When clang-format formats a file, it auto-detects the language using the file name. When formatting standard input or a file that doesn’t have the extension corresponding to its language, -assume-filename= option can be used to override the file name clang-format uses to detect the language.

Reply:
That doesn't really explain why we didn't initially do the format though, although I committed through VSCode so I didn't see if the precommit hook spit out anything.